suggestions.go

```diff
@@ -0,0 +1,75 @@
+package cli
+
+import (
+	"fmt"
+
+	"github.com/antzucaro/matchr"
+)
+
+const didYouMeanTemplate = "Did you mean '%s'?"
+
+func (a *App) suggestFlagFromError(err error, command string) (string, error) {
+	flag, parseErr := flagFromError(err)
+	if parseErr != nil {
+		return "", err
+	}
+
+	flags := a.Flags
+	if command != "" {
+		cmd := a.Command(command)
+		if cmd == nil {
+			return "", err
+		}
+		flags = cmd.Flags
+	}
```

Comment:
⭐⭐ _(seriously consider)_ consider also adding global flags to the command flags?

Reply:
I'm not sure If I get this correctly, but we don't have a distinction between global and non-global flags any more in v2, right? (so I'd go for `VisibleFlags()`).

Reply:
Oh, good question! I think so? I haven't actually had the chance to use v2 meaningfully, so I'm not 100% sure 😅

Reply:
Yeah I think we have no real support for flags added to every command, so the user has to specify it manually which should work fine with the current implementation. :)